producer-kafka/src/main/java/io/stargate/producer/kafka/KafkaProducerActivator.java

```diff
@@ -52,6 +54,15 @@ protected ServiceAndProperties createService() {
       CDCProducer producer =
           new KafkaCDCProducer(
               metrics.get().getRegistry(KAFKA_CDC_METRICS_PREFIX), configStore.get());
+      try {
+        producer.init().get();
```

Comment:
Why do we need to return a `CompletableFuture` from `init()` if we wait for it to complete anyway?

Reply:
Yes, it would be good enough for this producer, but other producers that implement the interface may want to provide an async init and let the caller decide.

Reply:
I'm wondering, should we `init` it at activations time or let the CDC service call `init`?

Reply:
That was my initial idea but I moved it to the extension itself because of the classpath problem.